content/04.network_applications.md

```diff
@@ -39,5 +39,27 @@
 
-1. Can mention EHR use and other related applications
-2. Mention Tiffany's work on private data embeddings
+Using knowledge graphs for clinical applications is in early stages of development and these graphs are used to improve patent healthcare.
+Typically, these graphs are constructed from electronic health records (EHR) where nodes represent patients, drugs and diseases and edges represent relationships such as a patient being perscribed a treatment or a patient being diagnosed with a disease [@pmid:26306276; @doi:10.1145/2110363.2110415; @arxiv:1707.05340; @doi:10.1186/s12859-015-0549-5].
+Tasks in this context range from improving patient diagnoses [@doi:10.1109/TKDE.2016.2605687;@arxiv:1709.06908] to recommending safer drugs for patients [@arxiv:1710.05980; @arxiv:1709.06908].
+We discuss approaches that project knowledge graphs into a low dimensional space to make clinical predictions.
+
+Early work in this field used translational models (Figure {@fig:unifying_techniques_overview} (b)) on knowledge graphs to make safe drug recommendations.
+Wang et al. [@arxiv:1710.05980] used a variant of the TransH [@raw:wangtransH] model to create a recommendation system for to recmmend safe drugs to patients.
+The authors constructed a disease-patient-drug network by integrating a patient-disease bipartite network with a patient-drug bipartite network.
+Every node in the graph was embedded while satisfying the following equation: $\textbf{h} - r \approx \textbf{t}$.
+Following the embedding step, the authors formulated their own similarity metric that selected drug combinations with a low number of interactions [@arxiv:1710.05980].
+Researchers in [@arxiv:1909.00672] had a similar variant of the TransH model on a medical knowledge graph, but they evaluated their model link prediction rather than patient recommendation.
+A limitation for both methods is that the ehr records are missing data for patients, which results in networks missing information as well.
+
+Deep learning techniques in this field have skipped over the node2vec and autoencoder models and started to use graph attention models [@arxiv:1706.03762].
+These models mimic the popular machine translation models [@arxiv:1409.0473] and are able to simultaneously embed knowledge graphs into a low dimensional space and perform the task at hand.
+Choi et al. [@arxiv:1611.07012] used a graph attention model to make patient diagnosis predictions.
+The authors constructed a directed graph using medical concepts from patient EHR data.
+This directed graph was fed into a graph attention network and then used to predict a patient's likelihood of heart failure [@arxiv:1611.07012].
+Other approaches have begun to adapt these graph attention models to perorm clinical tasks such as drug safety recommendations [@doi:10.1609/aaai.v33i01.33011126] and patient diagnoses [@arxiv:1906.04716].
+Similar to the transH approaches these methods are limited by missing data in the EHR records.
```

Comment:
If this applies to everything in the section, remove it from the individual paragraphs and perhaps deal with it in the paragraph below.